lib/bloomfilter/redis.rb

```diff
@@ -68,7 +68,7 @@ def stats
       def indexes_for(key)
         indexes = []
         @opts[:hashes].times do |i|
-          yield Zlib.crc32("#{key}:#{i+@opts[:seed]}") % @opts[:size]
+            yield Zlib.crc32("#{key}:#{i.to_s+@opts[:seed]}") % @opts[:size]
```

Comment:
But that's correct.. both `seed` and `i` are integers, no?

https://github.com/igrigorik/bloomfilter-rb/blob/8d64e23b0e0678272e70a0cd0c27db3f92678c6e/lib/bloomfilter/redis.rb#L8